sdk/storage/Azure.Storage.Common/src/Sas/AccountSasResourceTypes.cs

```diff
@@ -2,108 +2,80 @@
 // Licensed under the MIT License.
 
 using System;
-using System.ComponentModel;
 using System.Text;
 
 namespace Azure.Storage.Sas
 {
     /// <summary>
     /// Specifies the resource types accessible from an account level shared
-    /// access signature.  Use <see cref="ToString"/> to produce a value that
-    /// can be used for <see cref="AccountSasBuilder.ResourceTypes"/>.
+    /// access signature.
     /// </summary>
-    public struct AccountSasResourceTypes : IEquatable<AccountSasResourceTypes>
+    [Flags]
+    public enum AccountSasResourceTypes
     {
         /// <summary>
-        /// Gets a value indicating whether service-level APIs are accessible
+        /// Indicates whether service-level APIs are accessible
         /// from this shared access signature (e.g., Get/Set Service
         /// Properties, Get Service Stats, List Containers/Queues/Tables/
         /// Shares).
         /// </summary>
-        public bool Service { get; set; }
+        Service = 1,
 
         /// <summary>
-        /// Gets a value indicating whether blob container-level APIs are accessible
+        /// Indicates whether blob container-level APIs are accessible
         /// from this shared access signature (e.g., Create/Delete Container,
         /// Create/Delete Queue, Create/Delete Table, Create/Delete Share, List
         /// Blobs/Files and Directories).
         /// </summary>
-        public bool BlobContainer { get; set; }
+        BlobContainer = 2,
```

Comment:
I guess I totally missed this in your big rename since it touched so much, but this needs to be just `Container`.  It's a generic name that refers to `BlobContainer`, `Queue`, and `Share` across Blobs, Queues, and Files.

Reply:
Fixed